endoreg_db/services/lookup_store.py

```
@@ -44,3 +44,74 @@ def set(self, key: str, value: Any, ttl: int = DEFAULT_TTL_SECONDS) -> None:
+        required_fields = ['patient_examination_id', 'requirementsBySet', 'requirementStatus', 'available_examinations']
+        missing_fields = [field for field in required_fields if field not in data]
+        
+        if missing_fields:
+            import logging
+            logger = logging.getLogger(__name__)
+            logger.warning(f"Missing fields in lookup data for token {token}: {missing_fields}")
+            
+            # Try to recover patient_examination_id from token or related data
+            if 'patient_examination_id' in missing_fields:
+                # Attempt to extract from token or find related examination
+                recovered_id = self._recover_patient_examination_id(token)
```

Pylint (3.3.7)

[error] Assigning result of a function call, where the function returns None

(E1128)